purview/models/base.py

```diff
@@ -183,7 +183,17 @@ def inflate_results(results, resources_to_expand):
                     else:
                         serialized_node = inflated_node.serialized
 
-                    serialized_node['resource_type'] = node_label
+                    if node_label == 'KojiBuild' and 'type_' in serialized_node:
```

Comment:
This whole if statement could be reduced by overriding the `node_label` variable above when it is a `KojiBuild` of type `container`.